### PR TITLE
BE-1501: added audiences support for fulfilment models

### PR DESCRIPTION
**Pull request by slava-petrov**

### CODE REVIEW

File: livestyled/models/fulfilment_point.py [hunk 1]
```diff
@@ -100,6 +101,15 @@ def __init__(
         self.reference = reference
         self.external_id = external_id
 
+        audience_iris = []
+        for audience in audiences:
```
Comment (tomrutherford-livestyled) [inline livestyled/models/fulfilment_point.py:105]: I don't think this is required, its a half attempt at field validation

I could still call this function with [12, 'user_management/audiences/2', 'somerandomiri/13'] and that would be valid. Lets just make the function require an array of iri's and not do any specific formatting.

**Review by tomrutherford-livestyled — CHANGES_REQUESTED** [review]

File: livestyled/models/fulfilment_point.py [hunk 2]
```diff
@@ -99,6 +100,7 @@ def __init__(
         self.position = position
         self.reference = reference
         self.external_id = external_id
+        self.audiences = audiences
```
Comment (ndy40) [inline livestyled/models/fulfilment_point.py:103]: I think you need to do the same thing like `categories` below.

Comment (ndy40) [reply livestyled/models/fulfilment_point.py:103]: You also need to update the `schema/fulfilment_point.py`, the Audience needs a schema like 
`categories = RelatedResourceField(schema=FulfilmentPointCategorySchema, many=True, missing=[])`. You need to attache an AudienceSchema to it.

**Review by ndy40 — CHANGES_REQUESTED** [review]

File: livestyled/schemas/fulfilment_point.py [hunk 3]
```diff
@@ -61,3 +62,4 @@ class Meta:
     categories = RelatedResourceField(schema=FulfilmentPointCategorySchema, many=True, missing=[])
     venue = RelatedResourceLinkField(schema=VenueSchema, required=False, missing=None)
     external_id = fields.String(missing=None, data_key='externalId')
+    audiences = RelatedResourceField(schema=AudienceSchema, many=True, missing=[])
```
Comment (ndy40) [inline livestyled/schemas/fulfilment_point.py:65]: Have you tested this? I thought this should be `RelatedResourceLinkField`  like the others. please write a test to make sure this works.

**Review by ndy40 — CHANGES_REQUESTED** [review]

**Review by tomrutherford-livestyled — APPROVED** [review]

**Pull request merged**